src/UI/Components/Diagram/Canvas.tsx

```diff
@@ -21,6 +24,13 @@ const Canvas = ({
   mainServiceName: string;
   instance?: InstanceWithReferences;
 }) => {
+  const { environmentHandler } = useContext(DependencyContext);
+  const environment = environmentHandler.useId();
+  const baseUrlManager = new PrimaryBaseUrlManager(
+    globalThis.location.origin,
+    globalThis.location.pathname,
+  );
+  const baseUrl = baseUrlManager.getBaseUrl(process.env.API_BASEURL);
```

Comment:
I believe this is available in the DependencyContext. 
You should find it with ` urlManager.getApiUrl()` if you add the urlManager to line 27.